xbmc/interfaces/json-rpc/SystemOperations.cpp

```diff
@@ -95,6 +96,21 @@ JSONRPC_STATUS CSystemOperations::Reboot(const std::string &method, ITransportLa
     return FailedToExecute;
 }
 
+JSONRPC_STATUS CSystemOperations::InhibitIdleShutdown(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+    if ((parameterObject["inhibit"].isString() &&
+         parameterObject["inhibit"].asString().compare("toggle") == 0) ||
+        (parameterObject["inhibit"].isBoolean() &&
+         parameterObject["inhibit"].asBoolean() != g_application.IsIdleShutdownInhibited()))
+    {
+        CApplicationMessenger::GetInstance().SendMsg(TMSG_INHIBITIDLESHUTDOWN, !g_application.IsIdleShutdownInhibited());
```

Comment:
Too much indentation. And the curly braces aren't needed either (same in the `else if` block).